### ISSUE TITLE
Hide contextMenu when dragging on right-click

### ISSUE THREAD

**Issue by ElDeveloper** [swisnl/jQuery-contextMenu#391]
I was trying to get the menu to hide whenever the mouse was detected as being dragged under the right-click event, however I wasn't able to get this to work. It seems like all callbacks are overridden when the menu is being shown i.e. once the menu is shown I no longer receive callbacks for the `onmousemove` event. Is it possible to do this at all, or is this a technical restriction?

**Comment by bbrala** [comment]
Good question. Perhaps you'd have more luck with the drag events? Drag start and such

**Comment by ElDeveloper** [comment]
@bbrala, thanks for the suggestion! I believe you are referring to the [ondrag](http://www.w3schools.com/jsref/event_ondrag.asp) event? I tried this, but it didn't quite work, seems like the dragging events are restricted to "left-click" dragging only, not to right "click-dragging".

For now since we have to support both right click dragging and the context menu, we've made it such that shift has to be pressed for the context menu to be shown. If someone can think of other suggestions, we would welcome that a lot!
